Assets/Scripts/Game Awake Settings.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameAwakeSettings : MonoBehaviour
+{
+    [SerializeField] private SceneController sceneController;
+    [SerializeField] private EventManager eventManager;
+
+    private void Awake()
+    {
+
+
+        eventManager.Initialise();
+        sceneController.Initialise();
+
+        EventManager.instance.HandleOpenGame();
```

Comment:
if we implement some form for ScriptableSingleton framework then there will be no need to Initialize and scene controller can potentially just be dropped from this, as well as the serialized fields.

Reply:
It will also maybe give us some experience with.... a really difficult to debug problem when it comes to building that is really useful to know about and anticipate.

Reply:
This still needs to not call initialize, and Ideally should not be serialized

Reply:
Resolved